scan-irb/report-en.tex

```diff
@@ -260,11 +281,21 @@
   These results are provided as part of a research study.
 
 \item
-
+  %- if staff_observed == "false"
```

Comment:
Is this not repeating what starts on line 121? I've stared at that a couple of times and can't tell the difference. Sorry if I'm reading it wrong?

Reply:
Yes, this paragraph is duplicated in the template on master.

Reply:
Ahhhh.